### ISSUE TITLE
Error while parsing recursive ASN.1 specification

### ISSUE THREAD

**Issue by djelenc** [eerimoq/asn1tools#34]
I'm trying to parse the following recursive specification.

```
Example DEFINITIONS AUTOMATIC TAGS ::= BEGIN

Query ::= CHOICE {
  con Constraint,
  exp Expression
}

Constraint ::= SEQUENCE {
    operator  ENUMERATED {
      eq (0), ne (1), lt (2),
      le (3), gt (4), ge (5)
    },
    value     Value
}

Expression ::= SEQUENCE {
  operator  ENUMERATED { and (0), or (1) },
  left      Query,
  right     Query
}

Value ::= CHOICE {
  source  Entity,
  target  Entity,
  date    BinaryTime,
  service Service
}

Entity ::= PrintableString

Service ::= PrintableString

BinaryTime ::= INTEGER

END
```
When I run the `compile` command, I get an error.

```
$ compile messages.asn
error: cannot convert 'NoneType' object to bytes
```
Any help would be greatly appreciated.

**Comment by eerimoq** [comment]
Fixed in 0.125.1, available on PyPI.

**Comment by djelenc** [comment]
Hats off to you, Sir.
